symex.el

```diff
@@ -158,7 +159,9 @@ advises functions to enable or disable features based on user configuration."
   (cond ((eq symex-modal-backend 'hydra)
          (symex-hydra-initialize))
         ((eq symex-modal-backend 'evil)
-         (symex-evil-initialize))))
+         (symex-evil-initialize)))
+  ;; notify eldoc about symex motions.
+  (apply 'eldoc-add-command symex-motions))
```

Comment:
With the `symex-define-motion` macro, we shouldn't need this anymore, and can instead just have `(eldoc-add-command name)` in the macro itself (and can remove the list of `symex-motions` as well).

Reply:
Indeed ! good point :)